lib/_tls_wrap.js

```diff
@@ -453,6 +454,12 @@ TLSSocket.prototype._init = function(socket, wrap) {
   if (process.features.tls_npn && options.NPNProtocols)
     ssl.setNPNProtocols(options.NPNProtocols);
 
+  if (process.features.tls_alpn && options.ALPNProtocols) {
+    // keep reference in secureContext not to be GC-ed
+    ssl._secureContext.alpnBuffer = options.ALPNProtocols;
```

Comment:
This change comes from the old comment from @indutny  that

````
> +      w->alpn_protos_.Reset(args.GetIsolate(), args[0].As<Object>());
  What do you think about allocating and copying the data instead? I think I'm mostly against using Persistent for this reasons now.
````


I did not think of the reasons so much. I'm 0 for this change but I think we had better to have consistency between ALPN and NPN implementations.